_posts/2020-01-26-installing-blackfire-multiple-servers.md

```diff
@@ -0,0 +1,133 @@
+because it will work perfectly with the upcoming [Blackfire APM](https://hello.blackfire.io/apm).
+
+## The basics
+
+In the [Blackfire installation documentation](https://blackfire.io/docs/up-and-running/installation)
+you can read that Blackfire consists of 4 parts. First of course are the **Blackfire server**. 
+They receive all data and create a nice dashboard for you. 
+
+There is also an **Agent** than runs on our cloud that prepares and [cleans the data](https://blackfire.io/docs/reference-guide/faq#what-data-is-sent-to-the-blackfire-servers) 
+from sensitive information and sends it to the Blackfire servers. 
+
+Then there is a **PHP extension** that collects the metrics from your application
+and sends it to the **Agent**.
+
+Finally there is a **Client** that can start the profiling. There are two clients
+that you are most likely to use; a browser extension and a CLI tool. The client
+is used to make sure that only authorized users (you) can start profiling. 
+
+## Install
+
+When installing Blackfire on multiple servers, you do not need to have multiple instances
+of the Agent. You may have one agent for all your applications. You just need to
+install the PHP extension on all servers and configure the extension with the
+IP address to the machine that is running the Agent. 
+
+### Installing the agent
+
+Login to your [AWS Console](https://console.aws.amazon.com/), navigate to EC2
+and create a new instance. The first step is to select an AMI. Instead of selecting
+Ubuntu or something similar, search for "Blackfire" in the Community AMIs and select
+"Blackfire.io Agent".
+
+![Select AMI]({{ "/images/posts/blackfire/select-ami.png" | absolute_url }})
+
+Then it is time to select the instance type. The agent does not need much computer
+power, running with t2.micro is perfect. 
+
+![Select instance type]({{ "/images/posts/blackfire/select-instance-type.png" | absolute_url }})
+
+Review and launch your instance. Use a SSH key pair that you that you have access to
+because you will need to SSH into the instance. 
+
+![Review and Launch]({{ "/images/posts/blackfire/review.png" | absolute_url }})
+
+Now we wait a minute for the instance to be ready...
+
+When instance state is "running" we can SSH into the machine. You need the instance's
+public IP address for SSH, but note the private IP too. We will use it later. The 
+instance in the screenshot below has public IP 54.172.34.219 and private IP 172.31.84.248.
+
+![Instance example]({{ "/images/posts/blackfire/instance-show.png" | absolute_url }})
+
+When we have logged into the instance, we should update it and configure it with
+the server token related to your Blackfire account. You find the correct command 
+in the section "Configure the Agent" in the [Blackfire installation guide](https://blackfire.io/docs/up-and-running/installation). 
+
+![Login to agent instance]({{ "/images/posts/blackfire/login-to-agent.png" | absolute_url }})
+ 
+{% highlight cli %}
+sudo apt-get uppdate
```

Comment:
typo

Reply:
Ha, the only command I didnt not run myself..